README.md

```diff
@@ -52,8 +53,11 @@ import { DrilldownCalendar } from '@manufac/reactjs-calendar-heatmap';
   color={color}
   overview={overview}
   response={response}
+  showDayXAxisLabels={showDayXAxisLabels}
```

Comment:
Indentation

Reply:
In VS code and notepad++ the indentation is not there, don't know why it is appearing above here.
![image](https://user-images.githubusercontent.com/79472606/172986829-320228b1-d438-4be6-8f63-d006546854f8.png)
![image](https://user-images.githubusercontent.com/79472606/172987141-aa04e61e-6a60-4439-bc16-78ee5c21fb79.png)

Reply:
I  exchanged the position of  color and data attribute, It solved the problem.